modules/infra/src/main/scala/scaladex/infra/migrations/V7_2__edit_platform_and_language.scala

```diff
@@ -24,12 +24,10 @@ class V7_2__edit_platform_and_language extends BaseJavaMigration with ScaladexBa
       (for {
         oldArtifacts <- run(xa)(selectArtifact.to[Seq])
         groupedArtifacts = oldArtifacts.grouped(10000).toSeq
-        numberOfArtifactsUpdated <- groupedArtifacts
+        _ <- groupedArtifacts
           .map(artifacts => run(xa)(updatePlatformAndLanguage.updateMany(artifacts.map(_.update))))
           .sequence
-        _ = logger.info(s"Updated ${numberOfArtifactsUpdated.sum} artifacts")
```

Comment:
So we can change the code of migration files that were already executed?

Reply:
yes!

Reply:
I mean at least for the one with code!

Reply:
That's a good news! :slightly_smiling_face: